docs/data-sources/hosts.md

```diff
@@ -1,7 +1,7 @@
 ---
 # generated by https://github.com/hashicorp/terraform-plugin-docs
 page_title: "upcloud_hosts Data Source - terraform-provider-upcloud"
-subcategory: ""
+subcategory: Cloud
```

Comment:
Cloud category would include these non-product resources and data-sources. This could also be `Other`.

Reply:
Cloud seems reasonable to me.